Nodejs/Setup/BuildRelease.ps1

```diff
@@ -325,9 +325,12 @@ if ($name) {
 $version_file_backed_up = 0
 # Force use of a backup if there are pending changes to $version_file
 $version_file_force_backup = 0
-if (-not (tf status $version_file /format:detailed | Select-String "There are no pending changes.")) {
-    Write-Output "$version_file has pending changes. Using backup instead of tf undo."
-    $version_file_force_backup = 1
+$has_no_tf_workspace = (tf workspaces | Select-String -pattern "No workspace", "Unable to determine the workspace")
```

Comment:
The double negative is confusing from a readability standpoint, and we include the negation everywhere we use it, so it's better to negate it on assignment, so it can be ```if ($has_tf_workspace)``` everywhere else